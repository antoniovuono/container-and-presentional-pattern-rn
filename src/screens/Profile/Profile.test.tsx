import { render } from '@testing-library/react-native';
import { Profile } from '.';

describe('Profile Screen', () => {
  it('should be able to render profile screen correctly', () => {
    const tree = render(<Profile />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
