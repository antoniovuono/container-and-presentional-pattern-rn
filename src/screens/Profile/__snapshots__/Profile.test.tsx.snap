// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile Screen should be able to render profile screen correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#121214",
      "flex": 1,
      "gap": 20,
      "justifyContent": "center",
      "paddingHorizontal": 20,
    }
  }
>
  <Image
    source={
      {
        "uri": "https://static.vecteezy.com/ti/vetor-gratis/p1/23402465-homem-avatar-gratis-vetor.jpg",
      }
    }
    style={
      {
        "borderColor": "#FFFFFF",
        "borderRadius": 90,
        "borderWidth": 1.5,
        "height": 180,
        "width": 180,
      }
    }
  />
  <Text
    style={
      {
        "color": "#FFFFFF",
        "fontSize": 20,
        "fontWeight": "bold",
      }
    }
  >
    Nome do usuário
  </Text>
  <Text
    style={
      {
        "color": "#C4C4CC",
        "fontSize": 12,
        "textAlign": "center",
      }
    }
  >
    Biografia do usuário
  </Text>
  <TextInput
    autoCapitalize="none"
    onChangeText={[Function]}
    placeholder="username"
    placeholderTextColor="#C4C4CC"
    style={
      {
        "backgroundColor": "#202024",
        "borderRadius": 8,
        "color": "#FFFFFF",
        "padding": 15,
        "width": "100%",
      }
    }
  />
  <View
    accessibilityState={
      {
        "busy": undefined,
        "checked": undefined,
        "disabled": undefined,
        "expanded": undefined,
        "selected": undefined,
      }
    }
    accessibilityValue={
      {
        "max": undefined,
        "min": undefined,
        "now": undefined,
        "text": undefined,
      }
    }
    accessible={true}
    collapsable={false}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#00B37E",
        "borderRadius": 8,
        "justifyContent": "center",
        "opacity": 1,
        "padding": 15,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        {
          "color": "#fff",
          "fontSize": 12,
          "fontWeight": "bold",
        }
      }
    >
      Pesquisar
    </Text>
  </View>
</View>
`;
